### PR TITLE
New unit tests

### PR DESCRIPTION
**Pull request by Tejesh-SH**

### CODE REVIEW

File: ETWAnalyzer_uTest/EventDump/DumpFileTests.cs [hunk 1]
```diff
@@ -404,5 +405,116 @@ public void MinMaxTotalCount()
...
         }
 
+        
+        [Fact]
+        public void MinMaxTimeFilter()
+        {
+            KeyValuePair<string, MinMaxRange<decimal>>[] RangeValues = new KeyValuePair<string, MinMaxRange<decimal>>[]
+                {
+                    new KeyValuePair<string, MinMaxRange<decimal>>("1s-1000s", new MinMaxRange <decimal>(1, 1000)),
+                    new KeyValuePair<string, MinMaxRange<decimal>>("1000ms-2000ms", new MinMaxRange <decimal>(1000_000/ Million, 2000_000/Million)),
```
Comment (AloisKraus) [inline ETWAnalyzer_uTest/EventDump/DumpFileTests.cs:415]: 1000_000/Million = 1, 2000_000/Million is 2 so the value should be 1 and 2 right?

File: ETWAnalyzer_uTest/EventDump/DumpFileTests.cs [hunk 2]
```diff
@@ -404,5 +405,116 @@ public void MinMaxTotalCount()
...
+                    new KeyValuePair<string, MinMaxRange<decimal>>("1000ms-2000s", new MinMaxRange <decimal>(1000_000/ Million, 2000)),
```
Comment (AloisKraus) [inline ETWAnalyzer_uTest/EventDump/DumpFileTests.cs:416]: the first value should be also 1

**Review by AloisKraus — COMMENTED** [review]
Looks good. Some range values look strange but apart from that nice work!

File: ETWAnalyzer_uTest/EventDump/DumpFileTests.cs (hunk 1 above)
Comment (Tejesh-SH) [reply ETWAnalyzer_uTest/EventDump/DumpFileTests.cs]: corected

File: ETWAnalyzer_uTest/EventDump/DumpFileTests.cs (hunk 2 above)
Comment (Tejesh-SH) [reply ETWAnalyzer_uTest/EventDump/DumpFileTests.cs]: resolved

**Pull request merged**